test/images/agnhost/go.mod

```diff
@@ -0,0 +1,96 @@
+module k8s.io/kubernetes/test/images/agnhost
+
+go 1.17
+
+replace (
+	k8s.io/api => ../../../staging/src/k8s.io/api
+	k8s.io/apiextensions-apiserver => ../../../staging/src/k8s.io/apiextensions-apiserver
+	k8s.io/apimachinery => ../../../staging/src/k8s.io/apimachinery
+	k8s.io/apiserver => ../../../staging/src/k8s.io/apiserver
+	k8s.io/cli-runtime => ../../../staging/src/k8s.io/cli-runtime
+	k8s.io/client-go => ../../../staging/src/k8s.io/client-go
+	k8s.io/cloud-provider => ../../../staging/src/k8s.io/cloud-provider
+	k8s.io/cluster-bootstrap => ../../../staging/src/k8s.io/cluster-bootstrap
+	k8s.io/code-generator => ../../../staging/src/k8s.io/code-generator
+	k8s.io/component-base => ../../../staging/src/k8s.io/component-base
+	k8s.io/component-helpers => ../../../staging/src/k8s.io/component-helpers
+	k8s.io/controller-manager => ../../../staging/src/k8s.io/controller-manager
+	k8s.io/cri-api => ../../../staging/src/k8s.io/cri-api
+	k8s.io/csi-translation-lib => ../../../staging/src/k8s.io/csi-translation-lib
+	k8s.io/kube-aggregator => ../../../staging/src/k8s.io/kube-aggregator
+	k8s.io/kube-controller-manager => ../../../staging/src/k8s.io/kube-controller-manager
+	k8s.io/kube-proxy => ../../../staging/src/k8s.io/kube-proxy
+	k8s.io/kube-scheduler => ../../../staging/src/k8s.io/kube-scheduler
+	k8s.io/kubectl => ../../../staging/src/k8s.io/kubectl
+	k8s.io/kubelet => ../../../staging/src/k8s.io/kubelet
+	k8s.io/kubernetes => ../../../
+	k8s.io/legacy-cloud-providers => ../../../staging/src/k8s.io/legacy-cloud-providers
+	k8s.io/metrics => ../../../staging/src/k8s.io/metrics
+	k8s.io/mount-utils => ../../../staging/src/k8s.io/mount-utils
+	k8s.io/pod-security-admission => ../../../staging/src/k8s.io/pod-security-admission
+	k8s.io/sample-apiserver => ../../../staging/src/k8s.io/sample-apiserver
+	k8s.io/sample-cli-plugin => ../../../staging/src/k8s.io/sample-cli-plugin
+	k8s.io/sample-controller => ../../../staging/src/k8s.io/sample-controller
```

Comment:
these aren't maintained by the update-vendor.sh script, are they?

Reply:
I've copied pasted all the replaces from another place to be able to solve the imports and move forward, I have to curate this list